src/pybind/mgr/smb/resources.py

```diff
@@ -103,6 +103,22 @@ def convert(self, operation: ConversionOp) -> Self:
         return self
 
 
+@resourcelib.component()
+class QoSConfig(_RBase):
+    """Quality of Service configuration for CephFS shares."""
+
+    read_iops_limit: Optional[int] = None
+    write_iops_limit: Optional[int] = None
+    read_bw_limit: Optional[int] = None
```

Comment:
So we're going with `bw` after all? I still think it's better to spell out `bandwidth` since it's not *that* long, but I won't push back again.

Reply:
Fair enough.  Thanks for the input.